vendor/modules.txt

```diff
@@ -1614,6 +1614,7 @@ sigs.k8s.io/structured-merge-diff/v4/value
 # sigs.k8s.io/yaml v1.3.0
 ## explicit; go 1.12
 sigs.k8s.io/yaml
+# github.com/openshift/api => github.com/bpickard22/api v0.0.0-20240715224255-5ff1b2cdd555
```

Comment:
Need to point to api only when the open api PR gets merged to release-4.14. This comment is just for future reference.